src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -478,16 +447,66 @@ const myTeamsQuery = gql`query myTeamsQuery {
 	}
 }`;
 
+// Return a component importer function based on content group type from Contentful
+const getComponentFromType = type => {
+	switch (type) {
+		case 'mlCampaignHero':
+			return CampaignHero;
+		case 'mlLoanDisplay':
+			return CampaignLoanWrapper;
+		case 'monthlyGoodSelector':
+			return MonthlyGoodSelectorWrapper;
+		case 'frequentlyAskedQuestions':
+			return FrequentlyAskedQuestions;
+		case 'testimonialCards':
```

Comment:
This one along with as it's import can be removed. It's no longer selectable in prod or dev when creating a content group.